app/kumactl/cmd/install/testdata/install-control-plane.dump-values.yaml

```diff
@@ -395,8 +395,8 @@ kumactl:
     tag:
 
 kubectl:
-  # rancher maintains an image for all k8s versions */ } }
-  # see: https://hub.docker.com/r/rancher/kubectl */ } }
+  # kuma image that support v1.20.15 image */ } }
+  # see: https://hub.docker.com/r/kumahq/kubectl */ } }
```

Comment:
I've migrated this image but it looks like comment didn't get changed last time

Reply:
How is this image built then?

Reply:
https://github.com/kumahq/kuma/tree/master/tools/kubectl was introduced with arm64 support because rancher's image doesn't have binaries to run a script that we use.